spec/factories/ocn_resolution.rb

```diff
@@ -6,6 +6,5 @@
   factory :ocn_resolution, class: "Clusterable::OCNResolution" do
     deprecated { rand(1_000_000) }
```

Comment:
perhaps worth enforcing that `deprecated != resolved` just for that one-in-a-million time that is bound to happen more often than that out of spite

Reply:
Done. I am enforcing now in the factory that the OCNs are positive and that one is a positive amount greater than the other.